test/integration/check/test_check.py

```diff
@@ -0,0 +1,41 @@
+import os
+import re
+
+from utils import run_command_and_get_output
+
+
+def test_check_cmd(tt_cmd, tmpdir):
+    # testing with unset env variable
+    if os.environ.get("TT_CLI_INSTANCE") is not None:
+        del os.environ['TT_CLI_INSTANCE']
+    cmd = [tt_cmd, "check"]
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+    assert re.search(r"Failed to get instance path", output)
+
+    # testing with non-existent instance file
+    os.environ["TT_CLI_INSTANCE"] = 'non-existent-file'
+    cmd = [tt_cmd, "check"]
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+    assert re.search(r"No such file or directory", output)
+
+    # testing instance file with incorrect syntax
+    # temporary file is being created
+    with open('/tmp/incorrect_temp_file', 'w') as fp:
+        fp.write('do print(2+2) -- end')
```

Comment:
Just create several test applications in some `test_app` directory.

Reply:
Done. Created two separate lua files, which are copied to a temporary directory during tests